src/halmos/sevm.py

```diff
@@ -151,14 +183,21 @@ def instruction_length(opcode: Any) -> int:
     return (opcode - EVM.PUSH0 + 1) if EVM.PUSH1 <= opcode <= EVM.PUSH32 else 1
 
 def wextend(mem: List[Byte], loc: int, size: int) -> None:
-    if len(mem) < loc + size:
-        mem.extend([BitVecVal(0, 8) for _ in range(loc + size - len(mem))])
+    mem.extend([0] * (loc + size - len(mem)))
 
-def wload(mem: List[Byte], loc: int, size: int) -> Bytes:
+def wload(mem: List[Byte], loc: int, size: int, prefer_concrete=False) -> UnionType[bytes,Bytes]:
```

Comment:
prefer_concrete should hopefully be a temporary crutch -- I feel like this should be the default going forward but is a little risky now

Reply:
yeah, i also considered that, but i'm concerned that it might introduce too many unnecessary conversions between bv to int, back and forth, for some cases.  i think the current usage, limited to the codecopy-like cases, strikes a good balance.  let's expand its use when we encounter obvious cases that can improve performance!